tensorboard/plugins/profile/overview_page/overview-page.html

```diff
@@ -97,9 +97,9 @@
        text-decoration: underline;
      }
     </style>
-    <template is="dom-if" if="[[_show_error_message]]" restamp=true>
-      <div class="errorMessage"><span>[[_error_message]]</span></div></template>
-    <template is="dom-if" if="[[!_show_error_message]]" restamp=true>
+    <div class="errorMessage" hidden=[[!_show_error_message]]>
+      <span>[[_error_message]]</span></div>
```

Comment:
nit: can you put the closing div in a new line for legibility

Reply:
Done.